### PR TITLE
Added basic .travis.yml

### PR DESCRIPTION
**Pull request by fatimarafiqui**
travis.yml contains required php vesions, dependency installs and php linter

screen shot 2017-07-06 at 2 03 42 am

screen shot 2017-07-06 at 2 03 52 am

Build results of [Dummy PR](https://github.com/fatimarafiqui/FirstAide-web/pull/1)

PR create time: 02:05 Hrs IST

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+install:
+  - npm install
+  - composer install --no-interaction --prefer-source
+
+before_script:
+  - if find . -name "*.php" ! -path "./vendor/*" -exec php -l {} \; | grep "unexpected $end"; then exit 1; fi
+
+script:
+  - echo 'scripts'
```
Comment (npari) [inline .travis.yml]: What is the purpose of 'script' here?

Comment (fatimarafiqui) [inline .travis.yml]: This is where we will add phpunit test commands. [Reference](https://github.com/travis-ci-examples/php/blob/master/.travis.yml)

Comment (prathmeshranaut) [inline .travis.yml]: But you have to remove that scripts and replace it with PHP unit command.

Comment (npari) [inline .travis.yml]: @fatimarafiqui as of now it is doing nothing, so you can either remove it or add the command as suggested by Prathmesh.

Comment (fatimarafiqui) [inline .travis.yml]: @prathmeshranaut @npari  PHP unit module has not been pushed yet. So if we replace it with the command Travis will show error. The purpose of this is to test that travis has successfully reached to script-section in the logs. If you still suggest I can remove it directly.

**Pull request merged**